### PR TITLE
Improve cross-origin isolation related documents

### PR DESCRIPTION
**Pull request by agektmr**
Improves cross-origin isolation related documents

- https://web.dev/coop-coep
- https://web.dev/cross-origin-isolation-guide

Here's list of changes:

- Add report only mode step in the guide to recursively apply COEP to iframes.
- Add explanation about `allow="cross-origin-isolated"` to enable SharedArrayBuffer etc within an iframe.
- Add instructions how to check iframe and popup window's status of cross-origin isolation on DevTools.

### CODE REVIEW

**Comment by agektmr** [comment]
@lutzvahl can you review these changes?

**Comment by agektmr** [comment]
You can see the prevews at 
* https://deploy-preview-4693--web-dev-staging.netlify.app/coop-coep/
* https://deploy-preview-4693--web-dev-staging.netlify.app/cross-origin-isolation-guide/

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 1]
```diff
@@ -61,15 +61,24 @@ After you have determined which resources will be affected by cross-origin
...
+   resources, set [`Cross-Origin-Resource-Policy:
+   same-site`](https://resourcepolicy.fyi/#same-origin) header.
 3. Set the `crossorigin` attribute in the HTML tag on top-level document if the
    resource is served with [CORS](/cross-origin-resource-sharing/) (for example,
    `<img src="example.jpg" crossorigin>`).
-4. Make sure there are no cross-origin popup windows that require communication
+4. If cross-origin resources loaded into iframes involve another layer of
+   iframes, recursively apply steps described in this section before moving
+   forward.
+5. Once you confirm that all cross-origin resources are opt-in, set the
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:78]:
```suggestion
5. Once you confirm that all cross-origin resources are opted-in, set the
```

**Review by mihajlija — APPROVED** [review]

**Comment by kaycebasques** [comment]
@agektmr merge yourself when ready please

**Comment by lutzvahl** [comment]
LGTM content wise

**Pull request merged**